### PR TITLE
Remove `Http` prefix from `HttpClient`.

### PR DESCRIPTION
**Pull request by Nyholm**
This will replace https://github.com/php-http/fig-standards/pull/42

@dbu or @sagikazarmark, could you approve this again?

### CODE REVIEW

File: proposed/http-client/http-client.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ should not follow redirect nor throw exceptions on HTTP responses with status 4x
...
+### ClientInterface
 
 ```php
 namespace Psr\Http\Client;
 
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
 
-interface HttpClient
+interface Client
```
Comment (dbu) [inline proposed/http-client/http-client.md]: is PSR standard naming with Interface suffix or without? i think we said we do without suffix, but then the title of this section should be `### Client interface` i think.

Comment (Nyholm) [reply proposed/http-client/http-client.md]: Thank you

Comment (sagikazarmark) [reply proposed/http-client/http-client.md]: Ah, I was hoping noone notices :trollface:

Comment (michaelcullum) [reply proposed/http-client/http-client.md]: Will need to be `ClientInterface` yep

Comment (Nyholm) [reply proposed/http-client/http-client.md]: Thank you

**Pull request merged**